src/ape/options.py

```diff
@@ -1,6 +1,37 @@
 import click
+import click_logging
 
 from ape import networks
+from ape.logging import logger as logger
+from ape.utils import Abort
+
+
+class PluginHelper:
+    _logger = logger
+
+    def info(self, msg):
+        return self._logger.info(msg)
+
+    def warning(self, msg):
+        return self._logger.warning(msg)
+
+    def success(self, msg):
+        return self._logger.info(f"SUCCESS: {msg}")
+
+    @staticmethod
+    def abort(msg, base_error=None):
+        if base_error:
+            raise Abort(msg) from base_error
+        raise Abort(msg)
+
+
+def plugin_helper():
```

Comment:
This decorator both sets the logger in `click_logging` as well as makes it a pass decorator to enable having access to the class where it is needed, all in one go